wg-batch/annual-report-2024.md

```diff
@@ -2,32 +2,136 @@
 
 ## Current initiatives and Project Health
 
-
 1. What work did the WG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+See [2024 Highlights](#2024-highlights).
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+  Yes, JobSet has 1 active owner at the moment.
```

Comment:
Isn't JobSet active owners are you and @ ahg-g?

Reply:
Not in 2024. I did ping Jobset maintainer after seeing this question and I’ll be a maintainer in 2025. 🤷 .

Reply:
Oh, I see. Thank you for describing that!